docs/lightning/rtl.md

```diff
@@ -1,5 +1,83 @@
+
+### Funding your BTC Wallet
+
+In order to participate in the Lightning Network, you first need to fund your on-chain Bitcoin wallet. To do this, click on "On-chain" on the Navigation Panel on the left. Then click on the "Generate Address" button.
+
+This will generate a new address and corresponding QR code for you to use to fund your on-chain wallet.
+
+### Opening a Channel
+
+Before you can spend Bitcoin via the Lighting network, you first have to open a Lightning channel to another node.
+And, you must first connect to a chosen lightning node as a "peer" before you can open a channel to it.
+A good place to find potential peers/channels is [here](https://1ml.com/node?order=channelcount&active=true).
+
+When you have selected another lightning node to connect to, click on "Lightning" in the RTL navigation panel, then click on "Peers/Channels". Under the "Connections" heading, find and click on "Peers" as shown:
+
+![RTL Create Peer](/images/lightning/RTL/RTL_create_peer_comp.png "RTL Create Peer")
+
+Click on the "Add Peer" button, then:
+
+- Fill in the required information for your selected peer and close that window.
+- Click on the "Channels" button that is right under the "Connections" heading.
+- Click on the "Open Channel" button.
+- Select your chosen peer, fill in the amount of Bitcoin that you wish to open the channel with, and then click on "Open Channel"
+in the lower right corner of that window.
+
+You can verify the status of the channel via the Channels page. The channel will be pending until the transaction used to open the channel is confirmed.
+
+
+### Sending Bitcoin using RTL & Lightning
+
+When your funding transaction is confirmed on the blockchain you are now ready to send Bitcoin using lightning.
+You will need a payment invoice from the person you want to pay. Typically, either the payee will send you an invoice or you will copy it from a webpage.
+
+Go to the RTL navigation pane and click on "Lightning" and then "Transactions".
+
+![RTL Send Payment](/images/lightning/RTL/RTL_payments_comp.png "RTL Send Payment")
+
+Click on "Send Payment" and paste your copy of the invoice in the window that opens.
+All of the details of the payment should automatically be filled in. 
+Now, just click the "Send Payment" button and the payment should be completed within seconds!
+
+### Receiving Bitcoin using RTL and Lightning
+
+To receive Bitcoin in any lightning wallet, you must first establish what is called "inbound capacity".
+There are some simple and some more complex ways to do this, but they are beyond the scope of this guide.
+A good overview of how to establish inbound capacity can be found [here](https://gist.github.com/bretton/53bc511b6fdafef31951199dd25bbf88).
+
+## Specs
+
+- Github: https://github.com/ride-the-lighting/RTL
+- Installed location: As a Docker container with configuration files located at: /opt/mynode/RTL
+- How to access: http://mynode.local:3010/ or http://mynode.local:3011
```

Comment:
I realize now that the second port is for https.

I have to fix that in other places too.

Reply:
Oops...I never even gave that a thought.